### PR TITLE
Cross-tenant mailbox migration updates

### PR DESCRIPTION
**Pull request by tweekerz**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by JoeDavies-MSFT** [comment]
@tweekerz Can you select "Resolve conflicts" above and remove the old sections where they are marked in the markdown file? Thanks

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 1]
```diff
@@ -86,30 +86,40 @@ Prepare the source tenant:
...
+    | -ResourceTenantDomain                       | Source tenant domain, such as contoso\.onmicrosoft.com.                                                                                                                                                                                                                             | Required             |
+    | -ResourceTenantAdminEmail                   | Source tenant admin’s email address. This is the source tenant admin who will be consenting to the use of the mailbox migration application sent from the target admin. This is the admin who will receive the email invite for the application.                                    | Required             |
+    | -ResourceTenantId                           | Tenant ID of the Source tenant. For example, the [Azure AD tenant ID](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant) of contoso\.onmicrosoft.com tenant.                                                                 | Required             |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
    | -ResourceTenantId                           | Tenant ID of the Source tenant. For example, the [Azure AD tenant ID](/azure/active-directory/fundamentals/active-directory-how-to-find-tenant) of contoso\.onmicrosoft.com tenant.                                                                 | Required             |
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 2]
```diff
@@ -86,30 +86,40 @@ Prepare the source tenant:
...
+    | -ResourceGroup                              | Azure resource group name that contains or will contain the Key Vault.                                                                                                                                                                                                              | Required             |
+    | -KeyVaultName                               | Azure Key Vault instance that will store your mailbox migration application certificate/secret.                                                                                                                                                                                     | Required             |
+    | -CertificateName                            | Certificate name when generating or searching for certificate in key vault.                                                                                                                                                                                                         | Required             |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
    | -CertificateName                            | Certificate name when generating or searching for certificate in Key Vault.                                                                                                                                                                                                         | Required             |
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 3]
```diff
@@ -86,30 +86,40 @@ Prepare the source tenant:
...
+    | -CertificateSubject                         | Azure Key Vault certificate subject name, such as CN=contoso_fabrikam.                                                                                                                                                                                                              | Required             |
+    | -AzureResourceLocation                      | The location (Display Name) of the Azure resource group and key vault. You can run `Get-AzResourceProvider -ProviderNamespace Microsoft.KeyVault | Select-Object -ExpandProperty Locations -Unique` to see the available names.                                                     | Required             |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
    | -AzureResourceLocation                      | The location (Display Name) of the Azure resource group and Key Vault. You can run `Get-AzResourceProvider -ProviderNamespace Microsoft.KeyVault | Select-Object -ExpandProperty Locations -Unique` to see the available names.                                                     | Required             |
```

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 4]
```diff
@@ -181,14 +191,14 @@ The target admin setup is now complete!
...
+    | -ResourceTenantDomain             | Source tenant domain name, such as contoso\.onmicrosoft.com.                                                                                                                                                          |
+    | -ApplicationId                    | Azure application ID (GUID) of the application used for migration. Application ID available via your Azure portal (Azure AD, Enterprise Applications, app name, application ID) or included in your invitation email. |
+    | -TargetTenantDomain               | Target tenant domain name, such as fabrikam\.onmicrosoft.com.                                                                                                                                                         |
+    | -TargetTenantId                   | Tenant ID of the target tenant. For example, the [Azure AD tenant ID](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant) of fabrikam\.onmicrosoft.com tenant.  |
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
    | -TargetTenantId                   | Tenant ID of the target tenant. For example, the [Azure AD tenant ID](/azure/active-directory/fundamentals/active-directory-how-to-find-tenant) of fabrikam\.onmicrosoft.com tenant.  |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tweekerz, kindly commit the suggested changes.

**Comment by tweekerz** [comment]
@JoeDavies-MSFT @JohanFreelancer9 I have resolved as requested, ping me on teams if you need anything else. Thanks!!!

**Comment by JohanFreelancer9** [comment]
@tweekerz - It seems that my suggested changes (including changing links to relative links) have been marked as resolved but have not actually been committed. Could you please check and confirm? Many thanks.

**Comment by yogkumgit** [comment]
@tweekerz Please resolve this conflict.

**Comment by yogkumgit** [comment]
@tweekerz Please resolve.

**Comment by yogkumgit** [comment]
@tiburd

**Comment by tweekerz** [comment]
I have resolved all changes back in July there is no further actions I can take, the only option I have is to unresolve. Please push the changes. @tiburd @yogkumgit @JohanFreelancer9 @JoeDavies-MSFT

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@yogkumgit @yogkumgit - My suggestions have been marked as resolved, but they have not been committed. Unfortunately, I cannot commit them - they need to be committed by the PR author or person who merges the PR (to commit the change in its own commit, click "Commit suggestion").

**Pull request merged**